assets/enhancements.css

```
@@ -585,6 +585,14 @@ html.theme-transitioning *:after {
   border-top: 1.5px solid var(--line-strong) !important;
   box-shadow: 0 -8px 32px -16px rgba(0, 0, 0, 0.4) !important;
   color: var(--ink) !important;
+
+  transform: translateY(calc(100% - 15px));
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)